src/appleseed/renderer/modeling/aov/aovfactoryregistrar.h

```diff
@@ -65,14 +76,14 @@ class APPLESEED_DLLSYMBOL AOVFactoryRegistrar
     typedef AOVFactoryArray FactoryArrayType;
 
     // Constructor.
-    explicit AOVFactoryRegistrar(
-        const foundation::SearchPaths& search_paths = foundation::SearchPaths());
+    explicit AOVFactoryRegistrar();
```

Comment:
Remove `explicit` as it's no longer required. But note for the record that I wish that C++ had explicit by default! :)